pom.xml

```diff
@@ -11,6 +11,14 @@
         <developerConnection>scm:git:https://github.com/CruGlobal/aem-bom.git</developerConnection>
     </scm>
 
+    <distributionManagement>
+        <repository>
+            <id>central</id>
```

Comment:
This is ok, and it's the only way to use https://cruglobal.jfrog.io/cruglobal/maven-cru-private-releases-local/org/cru/maven-settings-template/1/settings.xml as-is. I think I want to add a third line to that settings.xml with the id 'cruglobal.jfrog.io'; I've put that in my personal settings.xml and at some point I'll put it in version 2 of the maven-settings-template. When I do, I may come back and update this repo.

Reply:
(I just feel like it's weird to reference 'central' here, when 'central' semantically doesn't mean where we're deploying. I know that the id is only used for auth details, but still...)

Reply:
Yeah, I got this auto-generated from jfrog.